internal/buffer/save.go

```diff
@@ -113,7 +114,17 @@ func openFile(name string, withSudo bool) (wrappedFile, error) {
 		}
 	}
 
-	return wrappedFile{writeCloser, withSudo, screenb, cmd, sigChan}, nil
+	return wrappedFile{name, writeCloser, withSudo, screenb, cmd, sigChan}, nil
+}
+
+func (wf wrappedFile) Truncate(size int64) error {
+	if wf.withSudo {
+		// we don't need to stop the screen here, since it is still stopped
+		// by openFile()
+		cmd := exec.Command(config.GlobalSettings["sucmd"].(string), "dd", "count=0", "if=/dev/zero", "of="+wf.name)
```

Comment:
We can drop `if=/dev/zero` (thus avoid unneeded assumption about the presence of /dev/zero)? It doesn't matter where we "read" 0 bytes from, so we can just read them from stdin.